### ISSUE TITLE
Error code in RStudio while running chunk for auxiliary regression in Lab4

### ISSUE THREAD

**Issue by wskamanda** [DS4PS/cpp-523-sum-2021#7]
Hello
I am trying to run the chunks for Lab4, but the chunk for estimating auxiliary regression kept showing this error below; 

Error in .stargazer.wrap(..., type = type, title = title, style = style,  : object 'm.auxiliary' not found

Is there anything I can do to fix it? Please help
Thanks 
William

**Comment by lecy** [comment]
Please review the discussion board etiquette: 

https://ds4ps.org/cpp-523-sum-2021/help/

This is not meant as a nit-picky request, but is a professional norm of communication within the field. 

To assess the problem we need to see the code that is producing the error. Which chunk are you getting stuck on specifically?

**Comment by lecy** [comment]
```r
m.full <- lm( SWB ~ ND + PSS, data=dat )
m.naive <- lm( SWB ~ ND, data=dat )
m.auxiliary <- # your model here
```
It can't find object **m.auxiliary** because you haven't specified it yet. You need to add the proper variables to create the auxiliary regression.

**Comment by kidistbetter105** [comment]
Am facing the problem?

**Comment by kidistbetter105** [comment]
m.auxiliary <- # your model here
Do I have to add the model in here?

**Comment by lecy** [comment]
Correct!

**Comment by kidistbetter105** [comment]
I did , but still the same . 
<- lm( ND ~ SWB, data=dat )

**Comment by lecy** [comment]
Did you assign the model to the new object called **m.auxiliary**? 

```
m.auxiliary <- lm( ND ~ SWB, data=dat )
```

**Comment by kidistbetter105** [comment]
yes!

Am I wrong?

**Comment by lecy** [comment]
That’s correct, should resolve the error you were getting. Are you still receiving an error?

**Comment by kidistbetter105** [comment]
yes ! am still receiving the Error.

**Comment by lecy** [comment]
Can you please share your code from the chunk and the error?

**Comment by kidistbetter105** [comment]
ok

**Comment by kidistbetter105** [comment]
<table style="text-align:center"><tr><td colspan="2" style="border-bottom:
1px solid black"></td></tr><tr><td style="text-align:left"></td><td>DV:
PSS</td></tr>
<tr><td></td><td colspan="1" style="border-bottom: 1px solid
black"></td></tr>
<tr><td style="text-align:left"></td><td>ND</td></tr>
<tr><td colspan="2" style="border-bottom: 1px solid
black"></td></tr><tr><td
style="text-align:left">SWB</td><td>0.03<sup>***</sup></td></tr>
<tr><td style="text-align:left"></td><td>(0.01)</td></tr>
<tr><td style="text-align:left"></td><td></td></tr>
<tr><td
style="text-align:left">Constant</td><td>5.04<sup>***</sup></td></tr>
<tr><td style="text-align:left"></td><td>(0.32)</td></tr>
<tr><td style="text-align:left"></td><td></td></tr>
<tr><td colspan="2" style="border-bottom: 1px solid
black"></td></tr><tr><td
style="text-align:left">Observations</td><td>389</td></tr>
<tr><td style="text-align:left">Adjusted
R<sup>2</sup></td><td>0.02</td></tr>
<tr><td colspan="2" style="border-bottom: 1px solid
black"></td></tr><tr><td style="text-align:left">Standard errors in
parentheses</td><td style="text-align:right"><sup>*</sup>p<0.1;
<sup>**</sup>p<0.05; <sup>***</sup>p<0.01</td></tr>

**Comment by kidistbetter105** [comment]
This is what I get when I run the chunk.

**Comment by lecy** [comment]
That’s correct - it’s stargazer output. If you knit the file it will look proper (it’s HTML code that creates the table).

if you want to preview output in the document change the stargazer argument to type=“text” and it will create a basic view. Just change it back to type=“html” before you knit.

**Comment by kidistbetter105** [comment]
Ok now I understand! 
Thanks for the help!
